package.json

```diff
@@ -21,6 +21,9 @@
     "node": ">= 6.0.0",
     "npm": ">= 3.8.1"
   },
+  "files": [
+    "dist/js/ui-components.js"
```

Comment:
I guess you don't want this .. if I'm reading the [docs](https://docs.npmjs.com/files/package.json#files) correctly, this would limit the npm package to just that one file